src/web/mjs/connectors/YoungJump.mjs

```diff
@@ -14,34 +14,18 @@ export default class YoungJump extends SpeedBinb {
     }
 
     async _getMangas() {
-        return [
-            {
-                id: 'free_uj/',
-                title: 'ウルトラジャンプ - ultra jump'
-            },
-            {
-                id: 'free_yj/',
-                title: 'ヤングジャンプ - young jump'
-            }
-        ];
+        const request = new Request('https://www.youngjump.world/yj-rest-apis/getBookInfo.php', this.requestOptions);
```

Comment:
Opening this URL in browser shows an empty list.

Reply:
Thats because people need to be logged. There is no more free content on the site.